Jenkinsfile

```diff
@@ -273,6 +275,40 @@ pipeline {
             }
           }
         }
+        stage('pytest tests') {
```

Comment:
Can we add a "sh 'printenv'" like on the other stages?
Also, I see this on the logs:
"[2021-07-26T12:24:36Z ERROR mayastor_csi::nodeplugin_grpc] gRPC server failed with error: transport error: error creating server listener: Address already in use (os error 98)"
Do the pytests clean up after themselves?

Reply:
They should cleanup after themselves yes. All the containers should be destroyed. Unfortuantely, I have seen a case before wher the mayastor_csi causes a kernel hang and cannot be killed. This is one of the reasons why we ultimately want to run the tests completely within a VM.

Reply:
Will add a sh printenv though

Reply:
Thinking about the varios ways in which a Jenkins pipeline may fail or be aborted. There is no guarantee that pytest will perform all the necessary teardown in the case of an abnormal termination. Need to investigate what control pytest gives us in such situations. In any case we probably should include a "post" stage in the Jenkins pipeline as a catch all to try and ensure that all necessary cleanup operations have been performed regardless of the test result. However, I think this is better done under a separate Jira.